pkg/api/validation/validation.go

```diff
@@ -1784,7 +1901,7 @@ func ValidateResourceRequirements(requirements *api.ResourceRequirements, fldPat
 				limitValue = quantity.MilliValue()
 			}
 			if limitValue < requestValue {
-				allErrs = append(allErrs, field.Invalid(fldPath, quantity.String(), "limit cannot be smaller than request"))
+				allErrs = append(allErrs, field.Invalid(fldPath, quantity.String(), "must be greater than or equal to `request`"))
```

Comment:
A nit. There is no field named `request`. so how about "must be greater than or equal to the counterpart in \`requests\`"?